angular/core/services/viewport_service.coffee

```diff
@@ -0,0 +1,10 @@
+angular.module('loomioApp').factory 'ViewportService', ($window) ->
+  new class ViewportService
+
+    width: ->
```

Comment:
I'm uncomfortable about taking one "width" which is an int and returning another which is a string representing some size.

I think i'd more more happy with 'size' rather than 'width'.. or even approximateSize.

how does that sound?